tests/local.rs

```diff
@@ -28,7 +28,33 @@ async fn find_and_click_link(mut c: Client, port: u16) -> Result<(), error::CmdE
     let new_url = c.current_url().await?;
     let expected_url = format!("http://localhost:{}/other_page.html", port);
     assert_eq!(new_url.as_str(), expected_url.as_str());
-    Ok(())
+
+    c.close().await
+}
+
+async fn iframe_switch(mut c: Client, port: u16) -> Result<(), error::CmdError> {
+    let url = sample_page_url(port);
+    c.goto(&url).await?;
+    // Go to the page that holds the iframe
+    c.find(Locator::Css("#iframe_page_id")).await?.click().await?;
+
+    c.find(Locator::Id("iframe_button")).await.expect_err("should not find the button in the iframe");
+    c.find(Locator::Id("root_button")).await?; // Can find the button in the root context though.
+
+    // find and switch into the iframe
+    let iframe_element = c.find(Locator::Id("iframe")).await?;
+    iframe_element.frame().await?;
```

Comment:
Reading this really makes me think that `frame()` should be renamed to `enter_frame()`. `frame()` makes me think that it's going to return a _handle_ to the frame, not switch to it.